File: run/scripts/channels.js
addRoute(GET, "/hello_polyglot_js/(?<name>.+)", async (req, args) => {
    print(`Hello, ${args.name}! I am JavaScript!`);
    await send("hello_py", args.name);
    return respond().finish();
});

addStatelessChannel("hello_js", name => print(`Hello, ${name}! I am JavaScript!`));

File: run/scripts/channels.py
def handle(req, args):
    print(f"Hello, {args.name}! I am Python!")
    send("hello_js", args.name)
    return respond().finish()


addRoute(GET, "/hello_polyglot_py/(?<name>.+)", handle)

addStatelessChannel("hello_py", lambda name: print(f"Hello, {name}! I am Python!"))
